src/app/AttributeCache.h

```diff
@@ -217,6 +216,10 @@ class AttributeCache : protected ReadClient::Callback
      */
     CHIP_ERROR Get(const ConcreteAttributePath & path, TLV::TLVReader & reader);
 
+    CHIP_ERROR GetVersion(EndpointId mEndpointId, ClusterId mClusterId, Optional<DataVersion> & aVersion);
+
+    static void SortFilterMap(std::map<DataVersionFilter, size_t> & aMap,
+                              std::vector<std::pair<DataVersionFilter, size_t>> & aVector);
```

Comment:
This does not seem to be implemented.

Reply:
yep, from early version, it has been removed, thanks